docs/developer_docs/javascript_testing.rst

```diff
@@ -105,7 +105,7 @@ Running Tests
 
 When you have written your tests, before you can run them, we need to bundle the Javascript for testing. In order to do this, from the root of the project run::
 
-    bin/kalite manage bundleassets
+    kalite manage bundleassets
```

Comment:
We shouldn't change this on the `developer_docs` pages, since it probably needs to be done this way when running from source (two more cases below).